charts/steward/values.yaml

```diff
@@ -84,6 +84,18 @@ pipelineRuns:
   logging:
     elasticsearch:
       indexURL: ""
+    forwarding:
+      # If "true" a forwarder will be used instead of sending logs directly to the destination
+      enabled: false
+      # The hostname of the forwarder
+      host: ""
+      # If "true", the fowarder is expected to run as a sidecare of the jenkinsfileRunner pod.
+      # In that case "host" will be ignored.
+      useSidecar: true
+      # The port the forwarder is listening to
+      port: '24224'
+      # The tag to use when sending data to the forwarder
+      tag: BUILD_LOGS
```

Comment:
Please add the documentation to the chart README.

Reply:
Done.